prov/util/src/util_av.c

```diff
@@ -1657,13 +1668,21 @@ struct util_cmap *ofi_cmap_alloc(struct util_ep *ep,
 	dlist_init(&cmap->peer_list);
 	fastlock_init(&cmap->lock);
 
+	cmap->cmd_queue = ofi_queue_alloc();
+	if (!cmap->cmd_queue)
+		goto err3;
+	cmap->cmd_write = cmap->cmd_read = 0;
+
 	if (pthread_create(&cmap->event_handler_thread, 0,
 			   cmap->attr.event_handler, ep)) {
 		FI_WARN(ep->av->prov, FI_LOG_FABRIC,
 			"Unable to create msg_cm_listener_thread\n");
-		goto err3;
+		goto err4;
 	}
+	cmap->cm_thread_state = UTIL_CM_THREAD_OPEN;
```

Comment:
`cm_thread_state` is set at several places but it's not read anywhere.

Reply:
This is needed for CMAP_EXIT handling. Will think how to avoid this state